packages/calypso-e2e/src/lib/blocks/block-flows/opentable.ts

```diff
@@ -44,6 +45,15 @@ export class OpenTableFlow implements BlockFlow {
 		await context.editorIframe.fill( selectors.searchInput, restaurant );
 		await context.editorIframe.click( selectors.suggestion( restaurant ) );
 		await context.editorIframe.click( selectors.embedButton );
+
+		// Wait until the block is fully rendered prior to next steps such as publishing.
+		// Alternatively, save as draft works here.
+		// See: https://github.com/Automattic/wp-calypso/issues/50302
+		const openTableFrameHandle = await context.editorIframe.waitForSelector(
+			`${ blockParentSelector } iframe`
+		);
+		const openTableFrame = ( await openTableFrameHandle.contentFrame() ) as Frame;
+		await openTableFrame.waitForSelector( selectors.openTableLogo );
```

Comment:
As noted in the code comments - an alternative is simply to perform `gutenbergEditorPaage.saveDraft`, which is the standard bypass method for https://github.com/Automattic/wp-calypso/issues/50302.